### PR TITLE
README: assorted copyediting

### PR DESCRIPTION
**Pull request by waldyrious**
I've tried to make some passages clearer for beginners, either by using links or by slightly rewording the sentences. The very first paragraph may look a bit strange; I felt that "take down" and "spoken word" could be slightly ambiguous, as they're both expressions that are (more commonly, I think) used to mean different things in other contexts, but I got the impression that you were deliberately avoiding the word "writing", which is why my proposed wording will probably sound a bit strange. I am of course willing to change the wording following what we discuss here.

The rest of the changes seem quite uncontroversial to me, but let me know if you'd like me to explain any of them better.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 ### What is Stenography?
 
-Stenography is the fastest way to input text. Court reporters have used machine stenography since the early 1900s to take down spoken word in realtime.
+Stenography is the fastest way to type down text. Court reporters have used machine stenography since the early 1900s to create written records of oral discourse in realtime.
```
Comment (panathea) [inline README.md]: Some stenographers (not me, personally) avoid equating typing to stenography. They actually *do* call it "writing" which is a little confusing within the context of the introduction.

> Stenography is the fastest way to write on a computer.

?

> create written records of human speech

could also work and may sound less scientific.

**Comment by panathea** [comment]
It all seems pretty good to me, definitely that opening line is the hardest.

This is unrelated to the PR content, but I should point out that I don't have a license specified for the text. I haven't been able to find a good resource for learning about licensing text…is there a copyleft license that is ideal for textbooks/documentation? I've been thinking of a creative commons license but I'm not sure what my rights are compared to an open source software project.

File: README.md (hunk 1 above)
Comment (waldyrious) [reply README.md]: I like both suggestions. The first one, in particular, qualifies "the fastest way", which was bothering me a bit. I'll make the changes right away.

**Comment by panathea** [comment]
Plover itself is GPLv2+, so I usually put related projects in the same family, the GPL free documentation license seems to cover this use case: https://www.gnu.org/licenses/fdl-1.3.en.html

I think that would probably be a good fit, I'm going to read more about it.

**Comment by panathea** [comment]
Okay, made a decision, I'm following Wikipedia and using CC BY-SA 4.0 International.

**Comment by waldyrious** [comment]
*(Note: you might want to use a slower pace in online discussions :) I'm publishing what I wrote for reference, and just in case it's still useful)*

If by copyleft you are explicitly talking about the "viral" (share-alike) property of the license, then [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/) is the obvious choice (for reference, you can read about Wikipedia's switch from the GFDL to CC-BY-SA [here](https://en.wikipedia.org/wiki/Wikipedia#Content_licensing) and [here](https://en.wikipedia.org/wiki/Wikipedia:Licensing_update)). I like to equate this to the GPL and other software licenses that include such a share-alike clause.

If you just mean an open license, then any of the [Open Definition-compliant](http://opendefinition.org/licenses/) Creative Commons licenses would do, the simplest choice being Attribution ([CC-BY](https://creativecommons.org/licenses/by/4.0/)), which I like to compare to the permissive software licenses like MIT.

Of course, IANAL, so I might recommend you to take a look at the [Creative Commons license chooser](https://creativecommons.org/choose/), or Github's [choosealicense.com](https://choosealicense.com/non-software/).

**Comment by panathea** [comment]
Yep, I realized I was going pretty fast. I'm glad to see that your message
reenforces the choice 👍

**Comment by panathea** [comment]
Thanks for the PR and help

**Comment by waldyrious** [comment]
My pleasure!

**Pull request merged**